src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -178,8 +186,11 @@ export const PatientInfoTable = ({
   });
 
   const handleOpenDetails = (patient) => {
-    history.push(`/events/${eventId}/ccps/${ccpId}/open/${patient.id}`);
```

Comment:
no, we just made it so that the page does not redirect when you open a dialog from clicking on a patient row. The /open URL is still used when the dialog is accessed from a snackbar link, such as when a patient is transported.